catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/complex/CmetaaAttribute.java

```diff
@@ -0,0 +1,1764 @@
+      new CmetaaAttribute(
+          RELATED_TRES,
+          "RELATED_TRES",
+          tre -> TreUtility.convertToInteger(tre, "RELATED_TRES"),
+          BasicTypes.INTEGER_TYPE);
+
+  public static final CmetaaAttribute ADDITIONAL_TRES_ATTRIBUTE =
+      new CmetaaAttribute(
+          ADDITIONAL_TRES,
+          "ADDITIONAL_TRES",
+          tre -> TreUtility.getTreValue(tre, "ADDITIONAL_TRES"),
+          BasicTypes.STRING_TYPE);
+
+  public static final CmetaaAttribute RD_PRC_NO_ATTRIBUTE =
+      new CmetaaAttribute(
+          RD_PRC_NO,
+          "RD_PRC_NO",
+          tre -> TreUtility.getTreValue(tre, "RD_PRC_NO"),
+          BasicTypes.STRING_TYPE);
+
+  public static final CmetaaAttribute IF_PROCESS_ATTRIBUTE =
+      new CmetaaAttribute(
+          IF_PROCESS,
+          "IF_PROCESS",
+          tre -> TreUtility.getTreValue(tre, "IF_PROCESS"),
+          BasicTypes.STRING_TYPE);
+
+  public static final CmetaaAttribute RD_CEN_FREQ_ATTRIBUTE =
+      new CmetaaAttribute(
+          RD_CEN_FREQ,
+          "RD_CEN_FREQ",
+          tre -> TreUtility.getTreValue(tre, "RD_CEN_FREQ"),
+          BasicTypes.STRING_TYPE);
+
+  public static final CmetaaAttribute RD_MODE_ATTRIBUTE =
+      new CmetaaAttribute(
+          RD_MODE,
+          "RD_MODE",
+          tre -> TreUtility.getTreValue(tre, "RD_MODE"),
+          BasicTypes.STRING_TYPE);
+
+  public static final CmetaaAttribute RD_PATCH_NO_ATTRIBUTE =
+      new CmetaaAttribute(
+          RD_PATCH_NO,
+          "RD_PATCH_NO",
+          tre -> TreUtility.getTreValue(tre, "RD_PATCH_NO"),
+          BasicTypes.STRING_TYPE);
+
+  public static final CmetaaAttribute CMPLX_DOMAIN_ATTRIBUTE =
+      new CmetaaAttribute(
+          CMPLX_DOMAIN,
+          "CMPLX_DOMAIN",
+          tre -> TreUtility.getTreValue(tre, "CMPLX_DOMAIN"),
+          BasicTypes.STRING_TYPE);
+
+  public static final CmetaaAttribute CMPLX_MAG_REMAP_TYPE_ATTRIBUTE =
+      new CmetaaAttribute(
+          CMPLX_MAG_REMAP_TYPE,
+          "CMPLX_MAG_REMAP_TYPE",
+          tre -> TreUtility.getTreValue(tre, "CMPLX_MAG_REMAP_TYPE"),
+          BasicTypes.STRING_TYPE);
+
+  public static final CmetaaAttribute CMPLX_LIN_SCALE_ATTRIBUTE =
+      new CmetaaAttribute(
+          CMPLX_LIN_SCALE,
+          "CMPLX_LIN_SCALE",
+          tre -> TreUtility.convertToFloat(tre, "CMPLX_LIN_SCALE"),
+          BasicTypes.FLOAT_TYPE);
+
+  public static final CmetaaAttribute CMPLX_AVG_POWER_ATTRIBUTE =
+      new CmetaaAttribute(
+          CMPLX_AVG_POWER,
+          "CMPLX_AVG_POWER",
+          tre -> TreUtility.convertToFloat(tre, "CMPLX_AVG_POWER"),
+          BasicTypes.FLOAT_TYPE);
+
+  public static final CmetaaAttribute CMPLX_LINLOG_TP_ATTRIBUTE =
+      new CmetaaAttribute(
+          CMPLX_LINLOG_TP,
+          "CMPLX_LINLOG_TP",
+          tre -> TreUtility.convertToInteger(tre, "CMPLX_LINLOG_TP"),
+          BasicTypes.INTEGER_TYPE);
+
+  public static final CmetaaAttribute CMPLX_PHASE_QUANT_FLAG_ATTRIBUTE =
```

Comment:
❓ I am not seeing this attribute or the next one per http://www.gwg.nga.mil/ntb/baseline/docs/stdi0002/stdi0002_v20/4Mar1999.pdf